utilities/openemr-cmd/openemr-cmd

```diff
@@ -1,16 +1,22 @@
 #!/bin/bash
+
+# Exit on error, unset var, pipe failure
+set -euo pipefail
```

Comment:
Does anyone have thoughts about adding `-x` to this? I realize it can be noisy, but in my experience the only people who ever look at the output of a shell script actually want to know what's going on.

Reply:
It's very noisy, and I suggested adding that when debugging versus standard usage. It might be interesting to add a debug flag that would `set -x`. I was thinking about adding one of the standard `die_msg()` that would print more debugging info at some point.